src/components/ConfirmDialog/DeleteDialog.tsx

```diff
@@ -0,0 +1,21 @@
+import Dialog from '../Dialog/Dialog';
+import Button from '../Button/Button';
+
+import './DeleteDialog.css';
+
+type DeleteDialogProps = {
+    onClose: () => void;
+    onConfirm: () => void;
+};
+export const DeleteDialog = ({ onClose, onConfirm }: DeleteDialogProps) => {
+    return (
+        <>
+            <Dialog title='Delete movie' onClose={onClose}>
+                <div className='deleteDialogBody'>
+                    <span>Are you sure you want to delete this movie?</span>
+                    <Button title='Confirm' onClick={onConfirm}></Button>
+                </div>
+            </Dialog>
+        </>
+    );
+};
```

Comment:
Here no react fragments are needed as <Dialog></Dialog> is only child.
Also you don't need function body and return , as you just returning one element so is just like:
export const DeleteDialog =({ onClose, onConfirm }: DeleteDialogProps)=>(<Dialog>code here</Dialog>);